common/flatpak-remote.c

```diff
@@ -477,6 +480,54 @@ flatpak_remote_set_default_branch (FlatpakRemote *self,
   priv->local_default_branch_set = TRUE;
 }
 
+/**
+ * flatpak_remote_get_main_ref:
+ * @self: a #FlatpakRemote
+ *
+ * Returns the main ref of this remote, if set.
```

Comment:
I think it would be good to add a sentence here to explain the concept. Something like:

The main ref is the ref that an origin remote is created for.